config/routes.rb

```diff
@@ -60,6 +60,7 @@
   patch "/profile/edit/password", to: "users#update_password"
   namespace :merchant do
```

Comment:
Sweet! So this `patch "/", to "dashboard#update"` is the same as `/merchant/dashboard` because of the `namespace` syntax?

Reply:
It took me a little while to understand it at first, but the `patch '/'` part is actually just like `/merchant/`, and here it's calling the merchant dashboard controller `update` action.